src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -213,6 +213,10 @@ private function retrieveFormFieldDescription(
         AdminInterface $admin,
         string $field
     ): FieldDescriptionInterface {
+        /**
+         * @var FieldDescriptionInterface|null
+         *                                     NEXT_MAJOR: remove annotation. Return type of interface will be correct
```

Comment:
Indention looks wrong

Reply:
I've moved it up. The indentation was because of `php-cs-fixer`